`TypeError` on collection with pytest 7.0

Just tried running poliastro test suite with pytest 7.0.0rc1 and found this error:

```
py38-fast run-test: commands[1] | pytest -m 'not slow and not mpl_image_compare' --mypy -vv
============================================================================================================ test session starts =============================================================================================================
platform linux -- Python 3.8.10, pytest-7.0.0rc1, pluggy-1.0.0 -- /home/juanlu/Projects/LSF/poliastro/library/.tox/py38-fast/bin/python
cachedir: .tox/py38-fast/.pytest_cache
hypothesis profile 'default' -> database=DirectoryBasedExampleDatabase('/home/juanlu/Projects/LSF/poliastro/library/.hypothesis/examples')
Matplotlib: 3.5.0
Freetype: 2.6.1
rootdir: /home/juanlu/Projects/LSF/poliastro/library, configfile: setup.cfg, testpaths: src, tests
plugins: hypothesis-6.30.1, remotedata-0.3.2, mypy-0.8.1, doctestplus-0.11.1, mpl-0.13, cov-2.5.1
collected 1172 items / 1 error / 33 deselected / 1138 selected                                                                                                                                                                               

=================================================================================================================== ERRORS ===================================================================================================================
_____________________________________________________________________________________________________ ERROR collecting tests/conftest.py _____________________________________________________________________________________________________
.tox/py38-fast/lib/python3.8/site-packages/pytest_doctestplus/plugin.py:210: in collect
    module = self.config.pluginmanager._importconftest(
E   TypeError: _importconftest() missing 1 required positional argument: 'rootpath'

```

It doesn't happen with pytest 6.2.5.

--- a/.github/workflows/python-tests.yml
+++ b/.github/workflows/python-tests.yml
@@ -44,6 +44,10 @@ jobs:
           - os: macos-latest
             python-version: 3.8
             toxenv: py38-test-pytestdev
+          - os: ubuntu-latest
+            python-version: '3.10'
+            toxenv: py310-test-pytest70
+            toxargs: --pre
 
     steps:
     - uses: actions/checkout@v2
@@ -56,7 +60,7 @@ jobs:
     - name: Install Tox
       run: python -m pip install tox
     - name: Run Tox
-      run: tox -v -e ${{ matrix.toxenv }}
+      run: tox ${{ matrix.toxargs }} -v -e ${{ matrix.toxenv }}
 
     # - name: Slack Notification
     #   uses: 8398a7/action-slack@v3

--- a/CHANGES.rst
+++ b/CHANGES.rst
@@ -1,5 +1,7 @@
-0.12.0 (unreleased)
+0.11.2 (unreleased)
 ===================
+
+- Fix version check for pytest 7.0.0rc1. [#171]
 
 0.11.1 (2021-11-16)
 ===================

--- a/pytest_doctestplus/plugin.py
+++ b/pytest_doctestplus/plugin.py
@@ -23,8 +23,9 @@ from .output_checker import (FIX, IGNORE_WARNINGS, REMOTE_DATA, SHOW_WARNINGS,
 _pytest_version = Version(pytest.__version__)
 PYTEST_GT_5 = _pytest_version > Version('5.9.9')
 PYTEST_GE_5_4 = _pytest_version >= Version('5.4')
-PYTEST_GE_6_3 = _pytest_version.is_devrelease or _pytest_version >= Version('6.3')
-PYTEST_GE_7_0 = _pytest_version.is_devrelease or _pytest_version >= Version('7.0')
+PYTEST_GE_7_0 = any([_pytest_version.is_devrelease,
+                     _pytest_version.is_prerelease,
+                     _pytest_version >= Version('7.0')])
 
 comment_characters = {
     '.txt': '#',
@@ -192,7 +193,7 @@ def pytest_configure(config):
         def collect(self):
             # When running directly from pytest we need to make sure that we
             # don't accidentally import setup.py!
-            if PYTEST_GE_6_3:
+            if PYTEST_GE_7_0:
                 fspath = self.path
                 filepath = self.path.name
             else:
@@ -206,9 +207,6 @@ def pytest_configure(config):
                     module = self.config.pluginmanager._importconftest(
                         self.path, self.config.getoption("importmode"),
                         rootpath=self.config.rootpath)
-                elif PYTEST_GE_6_3:
-                    module = self.config.pluginmanager._importconftest(
-                        self.path, self.config.getoption("importmode"))
                 elif PYTEST_GT_5:
                     module = self.config.pluginmanager._importconftest(
                         self.fspath, self.config.getoption("importmode"))
@@ -220,7 +218,7 @@ def pytest_configure(config):
                     if PYTEST_GT_5:
                         from _pytest.pathlib import import_path
 
-                    if PYTEST_GE_6_3:
+                    if PYTEST_GE_7_0:
                         module = import_path(fspath, root=self.config.rootpath)
                     elif PYTEST_GT_5:
                         module = import_path(fspath)
@@ -284,7 +282,7 @@ def pytest_configure(config):
     class DocTestTextfilePlus(pytest.Module):
 
         def collect(self):
-            if PYTEST_GE_6_3:
+            if PYTEST_GE_7_0:
                 fspath = self.path
                 filepath = self.path.name
             else:
@@ -475,7 +473,7 @@ class DoctestPlus(object):
         Skip paths that match any of the doctest_norecursedirs patterns or
         if doctest_only is True then skip all regular test files (eg test_*.py).
         """
-        if PYTEST_GE_6_3:
+        if PYTEST_GE_7_0:
             dirpath = Path(path).parent
         else:
             dirpath = path.dirpath()

--- a/setup.cfg
+++ b/setup.cfg
@@ -15,6 +15,7 @@ classifiers =
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
     Programming Language :: Python :: Implementation :: CPython
     Topic :: Software Development :: Testing
     Topic :: Utilities

--- a/tox.ini
+++ b/tox.ini
@@ -1,6 +1,6 @@
 [tox]
 envlist =
-    py{37,38,39}-test
+    py{37,38,39,310}-test
     codestyle
 requires =
     setuptools >= 30.3.0
@@ -19,6 +19,7 @@ deps =
     pytest60: pytest==6.0.*
     pytest61: pytest==6.1.*
     pytest62: pytest==6.2.*
+    pytest70: pytest==7.0.*
     pytestdev: git+https://github.com/pytest-dev/pytest#egg=pytest
 
 extras =

